app/app.js

```diff
@@ -42,8 +48,20 @@ function populateTemperFilter (chickenTemper){
   }
 }
 
-allChickens.forEach(function(a){
-  $('#chickens').append(a.toHtml());
-  populateChickenFilter(a.title);
-  populateTemperFilter(a.temperament);
-});
+
+Chicken.fetchAll = function() {
+  if (localStorage.rawChickenData) {
+    Chicken.loadAll(JSON.parse(localStorage.rawChickenData));
+  } else {
+    $.getJSON( 'data/appData.json', function ( json ) {
+      console.log( 'JSON Data: ', json);
+      localStorage.setItem('rawChickenData', JSON.stringify(json));
+    });
```

Comment:
In this else block of code, after you retrieve the data, it is successfully being stored in localStorage, but you didn't put it into the array of all chickens for display in the app. Calling Chicken.loadAll here, with the appropriate data, should correct that.